### ISSUE TITLE
Focus cannot move to the remove button on people picker selected item

### ISSUE THREAD

**Issue by rebeccaballantyne** [OfficeDev/office-ui-fabric-react#4816]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: ODB Share

#### Describe the issue:

When the <- arrow is hit after multiple people have been selected from the people picker, focus moves to the entire people picker item. The item is not actionable, but the remove button is. Focus should move to the remove button.

**Comment by JasonGore** [comment]
Covered by bug 16 of #6259. Closing as duplicate.
